samcli/commands/build/command.py

```diff
@@ -94,21 +95,7 @@
     help="Environment variables json file (e.g., env_vars.json) to be passed to build containers.",
     cls=ContainerOptions,
 )
-@click.option(
-    "--build-image",
-    "-bi",
-    default=None,
-    multiple=True,  # Can pass in multiple build images
-    required=False,
-    help="Container image URIs for building functions/layers. "
-    "You can specify for all functions/layers with just the image URI "
-    "(--build-image public.ecr.aws/sam/build-nodejs18.x:latest). "
-    "You can specify for each individual function with "
-    "(--build-image FunctionLogicalID=public.ecr.aws/sam/build-nodejs18.x:latest). "
-    "A combination of the two can be used. If a function does not have build image specified or "
-    "an image URI for all functions, the default SAM CLI build images will be used.",
-    cls=ContainerOptions,
-)
+@build_image_option(cls=ContainerOptions)
```

Comment:
Just curiosity, this `ContainerOptions` are the same type for both build and sync, why do we need to parameterize it?

Reply:
It's because if I import this class on options.py will be a circular import, this class is present also on build file.